espymetrics/dal/sqlite_dal.py

```diff
@@ -13,25 +13,114 @@
 
 * Not human readable - Binary format makes it harder to read/grep
 * More complex - More logic = more work = more time/resources?
-
 """
 
+import os
+import sqlite3
+
+import boltons
```

Comment:
Another unused import, but this one is more concerning. Because it's a third-party package, this can really harm the reader's impression of the following code's reusability.

You should add [PyLint](http://www.pylint.org/) to your editor. PyLint will tell you about unused imports.